docs/next/modules/en/pages/overview/features.adoc

```diff
@@ -8,23 +8,17 @@ This section describes features and feature stages in {product_name}.
 |===
 | Feature | Helm Feature Name | Sub-Feature | Default | Stage 
 
-1.1+| *Day 2 Operations*
-1.1+| `day2operations`
-| `etcdBackupRestore`
-| disabled
-| alpha
-
-| *Add-on Provider for Fleet*
-| _None_
+| *Add-on Provider Fleet*
+| `addon-provider-fleet`
 | _None_
 | enabled
-| GA
+| beta
 
 | *Agent TLS Mode*
 | `agent-tls-mode`
 | _None_
-| enabled
-| beta
+| disabled
```

Comment:
Should this be set to [enabled](https://github.com/rancher/turtles/blob/e8f9a701e97d82928cbc4c925087ae610f972fb2/charts/rancher-turtles/values.yaml#L45)?